sdk/protocol-manager/tests/tests.spec.ts

```diff
@@ -26,57 +25,73 @@ async function createProtocolManagerContext(): Promise<ProtocolManagerContext> {
 }
 
 describe('playground', () => {
-  let ctx: ProtocolManagerContext
+  let ctx: IProtocolManagerContext
   beforeAll(async () => {
     ctx = await createProtocolManagerContext()
   })
 
   it('template/maker', async () => {
-    const makerPoolId: MakerPoolId = {
+    const makerPoolId = {
       protocol: {
         name: ProtocolName.Maker as const,
         chainInfo: ChainInfo.createFrom({ chainId: 1, name: 'Ethereum' }),
       },
       ilkType: ILKType.ETH_A,
       vaultId: '123',
     }
-    await protocolManager.getPool(makerPoolId, ctx)
-    //   console.log(`
-    // MAKER POOL
-    // ----------------
-    // Protocol: ${result.protocol.name}
-    // Chain: ${result.protocol.chainInfo.name}
-    // IlkType: ${JSON.stringify(result.poolId)}
-    // ${result.baseCurrency.toString()}}
-    // ${JSON.stringify(result.collaterals, null, 4)}
-    // `)
+
+    protocolManager.init(ctx)
+    const result = await protocolManager.getPool(makerPoolId)
+    console.log(`
+  MAKER POOL
+  ----------------
+  Protocol: ${result.protocol.name}
+  Chain: ${result.protocol.chainInfo.name}
+  IlkType: ${JSON.stringify(result.poolId)}
+  ${result.baseCurrency.toString()}}
+  ${JSON.stringify(result.collaterals, null, 4)}
+  `)
   })
 
   it('template/spark', async () => {
-    await sparkPlugin.getPool(
-      {
-        protocol: {
-          name: ProtocolName.Spark,
-          chainInfo: ChainInfo.createFrom({ chainId: 1, name: 'Ethereum' }),
-        },
-        emodeType: EmodeType.None,
+    const sparkPoolId: SparkPoolId = {
+      protocol: {
+        name: ProtocolName.Spark as const,
+        chainInfo: ChainInfo.createFrom({ chainId: 1, name: 'Ethereum' }),
       },
-      ctx,
-    )
-    //console.log(result)
+      emodeType: EmodeType.None,
+    } as SparkPoolId
+
+    protocolManager.init(ctx)
+    const result = await protocolManager.getPool(sparkPoolId)
+    console.log(`
```

Comment:
I'm not a fan of console.log in tests, as nobody is going to check this. It is much better to validate those values with `expect` I know the tests are going to happen in a different PR, but just wanted to set the expectation here. Let me know if you disagree :)

Reply:
They're just placeholder before real tests. Just used to print stuff out to aid development